docs/blog/2019-06-08-delicious-simplicity-case-study-part-1/index.md

```diff
@@ -3,8 +3,8 @@ title: "Delicious Simplicity ❤️ Gatsby: Building Happily Ever After"
 date: 2019-06-08
 author: Michelle Gienow
 tags:
-  - case study
-  - developer experience
```

Comment:
People talk about `developer-experience` as one of the things they like about Gatsby all the time–so that seems more intuitive to me as a tag than `building-sites-faster`. There are also parts of DX that aren't necessarily about "building sites faster", or "building faster sites" which is also what that brings to mind. Could we keep both tags?

Reply:
I think that's a good point, doesn't hurt to have both by any means.